docs/release-notes/2.39.0.md

```diff
@@ -0,0 +1,65 @@
+
+### Query Analytics - Meta information
+
+QAN is an important tool that can help filter data and identify problematic queries. However, it can be challenging to determine which services, nodes, and other labels are linked to these queries. PMM 2.39.0 now includes a metadata table for QAN to address this issue. This table contains information such as Node name, Node ID, Service name, Service Type, and user name for each selected query. With this feature, you can easily identify the services, nodes, and labels associated with your identified queries.
+
+To view the metadata table, navigate to the **Main** menu and click on **Query Analytics (QAN)**. This will bring you to the **Query Analytics** page. Next, select the desired query and expand the **Metadata** tab. The metadata for that specific query will then be displayed.
+
+![clouds](../_images/PMM_QAN_metadata.png)
+
+
+## Improvements
+
+- [PMM-6279](https://jira.percona.com/browse/PMM-6279) - [QAN]: Query metadata not visible in new UI
+- [PMM-10270](https://jira.percona.com/browse/PMM-10270) -[Backup Management]: Enhanced MongoDB Backup Management to support restoring to different replica set clusters with the same configuration.
+- [PMM-11999](https://jira.percona.com/browse/PMM-11999) - [Advisors]: Expose check family field in advisors API
+- [PMM-12257](https://jira.percona.com/browse/PMM-12257) - [Advisors]: The **Advisors** page now includes a new **Technology** column, showing the database type for each advisor in the table. This helps you easily differentiate between the MongoDB, MySQL and PostgreSQL	advisors.
+- [PMM-11637](https://jira.percona.com/browse/PMM-11637) - In PMM 2.39.0, we updated the interface to reflect the new PMM logo as per the new branding.
+- [PMM-11206](https://jira.percona.com/browse/PMM-11206) - [Alerting]: Enhanced the **Add rule template** dialog box to support YAML content for multiple alert templates. Alert templates added in bulk are displayed individually on the **Alert rule templates** page.
+ 
+## Components updates
+- [PMM-12070](https://jira.percona.com/browse/PMM-12070) - We have updated the deprecated [Boomtable plugin](https://github.com/percona/grafana-dashboards/tree/main/panels/yesoreyeram-boomtable-panel) to the Grafana table plugin.
+ 
+## Bugs Fixed
+
+- [PMM-12151](https://jira.percona.com/browse/PMM-12151) - [QAN]: Explain plan not displayed if database name is missing in the query
+- [PMM-10391](https://jira.percona.com/browse/PMM-10391) - PMM did not work with external clickhouse versions greater than 22.5.2. The issue has been resolved now.
+- [PMM-11658](https://jira.percona.com/browse/PMM-11658) - pmm-agent crashes simultaneously on all nodes
+- [PMM-11992](https://jira.percona.com/browse/PMM-11992) - Database name is ignored while adding any Postgres
```

Comment:
What was the impact of this "ignoring"?

Reply:
Users who configured their Postgres databases with names other than "Postgres" were unable to add them. The issue has been resolved now.